docs/01_nodeos/06_logging/01_logging-levels.md

```diff
@@ -64,3 +64,43 @@ Sample `logging.json`:
  ]
 }
 ```
+
+## Expected output of log levels
+
+* `error`
+  * Log output that likely requires operator intervention. Error level logging should 
+    be reserved for conditions that are completely unexpected or otherwise needs human
```

Comment:
or an indication of a software error like impossible value of an enum, out of boundary array access, null pointers... something pointing to a bug

Reply:
Thanks. Incorporated this into the description.